README.md

```diff
@@ -91,13 +91,18 @@ To enable the Soroban RPC server provide the following command line flags when s
 
 The Soroban RPC Server will be avaialble on port 8000 of the container, and the base URL path for Soroban RPC will be `http://<container_host>:8000/soroban/rpc`. This endpoint uses [JSON-RPC](https://www.jsonrpc.org/specification) protocol. Refer to example usages in [soroban-example-dapp](https://github.com/stellar/soroban-example-dapp).
 
+To enable Soroban diagnostic events provide the following command line flag when starting the container:
```

Comment:
do you think it's worthwhile to mention `diagnostics cause unmetered execution`, I didn't know that either until you mentioned it in the pr description.

Reply:
Good idea. I'll add a note.

Reply:
b188754